zqd/handler.go

```diff
@@ -40,8 +41,9 @@ func NewHandler(core *Core, logger *zap.Logger) http.Handler {
 	h.Handle("/space/{space}/subspace", handleSubspacePost).Methods("POST")
 	h.Handle("/search", handleSearch).Methods("POST")
 	h.HandleFunc("/version", func(w http.ResponseWriter, r *http.Request) {
+		version := api.VersionResponse{Version: core.Version}
 		w.Header().Add("Content-Type", "application/json")
-		json.NewEncoder(w).Encode(&Version)
+		json.NewEncoder(w).Encode(&version)
```

Comment:
Nit: Define version closer to its use. Or say `json.NewEncocder(w).Encode(&api.VersionResponse{Version: core.Version})`.